### PR TITLE
Add vendor configuration for Quantcast Measure

### PR DESCRIPTION
**Pull request by SmithKevin**
This is the pull request to add Quantcast Measure configuration to the built in vendors.

### CODE REVIEW

**Comment by SmithKevin** [comment]
Hi.   We have signed the corporate CLA under Quantcast.   Please let me know if we are required to do anything else.

Thanks,

**Comment by rudygalfi** [comment]
Added parallel intent to implement here: #1883

**Comment by rudygalfi** [comment]
cc @avimehta for review

File: examples/analytics.amp.html [hunk 1]
```diff
@@ -127,6 +127,19 @@
 </script>
 </amp-analytics>
 
+<!-- Quantcast tracking -->
+<amp-analytics type="quantcast">
+<script type="application/json">
+{
+  "vars": {
+    "pcode": "p-1234567890abc",
+    "labels": ["Example.label.1", "Example.label.2"]
```
Comment (rudygalfi) [inline examples/analytics.amp.html]: @avimehta Can a var value be an array?

**Comment by rudygalfi** [comment]
Thanks for the PR! Did a quick pass, but looks good.

File: examples/analytics.amp.html (hunk 1 above)
Comment (SmithKevin) [inline examples/analytics.amp.html]: @rudygalfi  This works as it is today.  It just gets treated as a single string.  However I do have an issue open here  #1886 which would add a small change in functionality for arrays.

**Comment by avimehta** [comment]
lgtm from my side. Please resolve conflicts and then @rudygalfi will merge this.

**Comment by avimehta** [comment]
actually one comment, the link in the docs does not mention AMP. It might be better for users if the page had some AMP related information. Or if the link was amp specific.

**Comment by SmithKevin** [comment]
Fixed conflict.  We don't have any specific amp docs up on the site yet.   Marketing is still going through them and they should be up sometime next week.   I have instead changed the link to point to our guides page where the docs will eventually be posted.

**Comment by rudygalfi** [comment]
LGTM. cc @avimehta for one last check

**Comment by avimehta** [comment]
lgtm. please squash the commits into a single one. https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

**Comment by SmithKevin** [comment]
Done

**Comment by avimehta** [comment]
@rudygalfi please merge. 

@SmithKevin Thanks for adding the support!

**Comment by rudygalfi** [comment]
Will merge once Travis is green.

**Pull request merged**